build/Common.prod.props

```diff
@@ -24,7 +24,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftPublicApiAnalyzersPkgVer)" Condition=" $(OS) == 'Windows_NT' and '$(EnablePublicApi)'!='false'" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftPublicApiAnalyzersPkgVer)" Condition=" $(OS) != 'Linux' and '$(EnablePublicApi)'!='false'" PrivateAssets="All" />
```

Comment:
I would consider even removing system condition completely. It can impact on build time, but should cover scenario when someone is using linux to develop.

@utpilla, any opinions on that?

Reply:
This was discussed in the SIG meeting yesterday. We were in favor of removing this condition completely and have it run for all the platforms. It shouldn't add much to CI time as the checks for different platforms should be run in parallel.